CMakeLists.txt

```diff
@@ -1,198 +1,247 @@
-cmake_minimum_required(VERSION 2.8.6)
+cmake_minimum_required(VERSION 3.5)
 project(rviz_visual_tools)
 
-if(NOT MSVC)
-  add_compile_options(-std=c++11)
-  # Warnings
-  add_definitions(-W -Wall -Wextra
-    #-Wcast-qual
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-W -Wall -Wextra
     -Wwrite-strings -Wunreachable-code -Wpointer-arith
     -Winit-self -Wredundant-decls
     -Wno-unused-parameter -Wno-unused-function)
 endif()
 
-set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+# set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
Why just comment it out? Either we need it or not. I added it a while back for "C++11 optimizations", are you sure its doesn't apply in ROS2?
https://github.com/PickNikRobotics/rviz_visual_tools/commit/7ce702cb90dfb81db00d38ffefeb4ac1ef1ef82c

Reply:
Seems harmless:
https://cmake.org/cmake/help/v3.5/variable/CMAKE_EXPORT_COMPILE_COMMANDS.html

But i don't remember why i added it